velox/docs/functions/spark/array.rst

```diff
@@ -0,0 +1,38 @@
+=============================
+Array Functions
+=============================
+
+.. js:function:: array_sort(array(E)) -> array(E)
+    :noindex:
+
+    Returns an array which has the sorted order of the input array(E). The elements of array(E) must
+    be orderable. Null elements will be placed at the end of the returned array. ::
+
+        SELECT array_sort(ARRAY [1, 2, 3]); -- [1, 2, 3]
+        SELECT array_sort(ARRAY [3, 2, 1]); -- [1, 2, 3]
+        SELECT array_sort(ARRAY [2, 1, NULL]; -- [1, 2, NULL]
+        SELECT array_sort(ARRAY [NULL, 1, NULL]); -- [1, NULL, NULL]
+        SELECT array_sort(ARRAY [NULL, 2, 1]); -- [1, 2, NULL]
+
+.. function:: in(x, element) -> boolean
+
+    Returns true if the array ``x`` equals to any value of ``element``. the elements and x must be same type.
+    Supports BOOLEAN, REAL, DOUBLE, BIGINT, VARCHAR, TIMESTAMP, DATE input types.
+
+.. function:: size(x) -> bigint
+
+    Returns the size of the array ``x``. The function returns null for null input
+    if legacySizeOfNull is set to false. Otherwise, the function returns -1 for null input.
+    With the default settings, the function returns -1 for null input.
+
+.. function:: sort_array(array(E)[, ascendingOrder]) -> array(E)
```

Comment:
same as above; document this as 2 separate functions

How is this function different from array_sort? Perhaps, clarify this in the doc to avoid confusion.

Reply:
Sorry, I didn't explain clearly.
array_sort(array(E))：sort in ascending order, Null elements will be placed at the **end** of the returned array.
sort_array(array(E))：Null elements will be placed at the **beginning** of the returned array.
sort_array(array(E), ascendingOrder):  Null elements will be placed at the beginning of the returned array in ascending
order or at the end of the returned array in descending order.